README.md

```diff
@@ -15,6 +15,214 @@ Build
 
     $ npm run dist
 
+Usage
+-----
+
+**Caution: everything in this section is still pure fiction.**
+
+* Every operation is performed locally;
+* Synchronization with server shall be ran explicitly.
+
+### Collection
+
+```js
+
+  import Cliquetis from "cliquet";
+
+  var db = new Cliquetis();
+  var articles = db.collection('articles');
+
+```
+
+The collection object has the following attributes:
+
+* **last_modified**: last synchronization timestamp, ``undefined`` if never sync'ed.
+
+> Synchronization timestamps are persisted in the device local storage.
+
+
+### List records
+
+```js
+
+  articles.all().then(result => {
+    {
+      data: [
+        {
+          id: "705b17be-e957-4c14-8f4c-86f8eaac29c0",
+          title: "foo"
+        },
+        {
+          id: "68e63131-3859-40cc-a4f7-b237ca179329",
+          last_modified: 1432222889336,
+          title: "Web page"
+        },
+      ]
+    }
```

Comment:
Maybe we should invent another way to describe that resulting data structure; here the code is invalid, doesn't make much sense and could confuse newcomers.

Also, we should probably attach a `last_modified` property to that result object, as described above.